render/jvm/src/main/scala/plotly/Plotly.scala

```diff
@@ -47,7 +47,7 @@ object Plotly {
     b ++= printer.render(layout.asJson)
     b ++= ";\n\n  Plotly.plot('"
     b ++= div.replaceAll("'", "\\'")
-    b ++= "', data, layout);\n"
+    b ++= "', data, layout, {editable: true, responsive: true, showEditInChartStudio: true, plotlyServerURL: 'https://chart-studio.plotly.com'});\n"
```

Comment:
I guess we should add a `plotly.Configuration` class, with these fields, then pass it around alongside `Layout`, so that users can enable / disable these.